tests/test_affect_history.py

```
@@ -0,0 +1,645 @@
+from pathlib import Path
+from nima_core.cognition.affect_history import (
+    AffectHistory,
+    AffectSnapshot,
+    AFFECTS,
+)
+
+
+class TestAffectSnapshot:
+    """Test AffectSnapshot dataclass functionality."""
+    
+    def test_snapshot_creation(self, sample_baseline):
```

Ruff (0.15.0)

[warning] Unused method argument: `sample_baseline`

(ARG002)